Possible flaky test_setup_exchanges_test

## Problem Definition

Seen in a Travis OSX run. https://travis-ci.org/github/rotki/rotki/jobs/679663462

```
=================================== FAILURES ===================================
_____________________________ test_setup_exchange ______________________________
rotkehlchen_api_server = <rotkehlchen.api.server.APIServer object at 0x116b287f0>
    def test_setup_exchange(rotkehlchen_api_server):
        """Test that setting up an exchange via the api works"""
        # Check that no exchanges are registered
        response = requests.get(api_url_for(rotkehlchen_api_server, "exchangesresource"))
        assert_proper_response(response)
        json_data = response.json()
        assert json_data['message'] == ''
        assert json_data['result'] == []
    
        # First test that if api key validation fails we get an error, for every exchange
        for name in SUPPORTED_EXCHANGES:
            data = {'name': name, 'api_key': 'ddddd', 'api_secret': 'fffffff'}
            if name == 'coinbasepro':
                data['passphrase'] = '123'
            response = requests.put(
                api_url_for(rotkehlchen_api_server, "exchangesresource"), json=data,
            )
            assert_error_response(
                response=response,
                contained_in_msg=[
                    'Provided API Key or secret is invalid',
                    'Provided API Key is invalid',
                    'Provided API Key is in invalid Format',
                    'Provided API Secret is invalid',
                    'Provided Gemini API key needs to have "Auditor" permission activated',
                ],
>               status_code=HTTPStatus.CONFLICT,
            )
rotkehlchen/tests/api/test_exchanges.py:70: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
response = <Response [409]>
contained_in_msg = ['Provided API Key or secret is invalid', 'Provided API Key is invalid', 'Provided API Key is in invalid Format', 'Provided API Secret is invalid', 'Provided Gemini API key needs to have "Auditor" permission activated']
status_code = <HTTPStatus.CONFLICT: 409>, result_exists = False
    def assert_error_response(
            response: Optional[requests.Response],
            contained_in_msg: Optional[Union[str, List[str]]] = None,
            status_code: HTTPStatus = HTTPStatus.BAD_REQUEST,
            result_exists: bool = False,
    ):
        assert (
            response is not None and
            response.status_code == status_code and
            response.headers["Content-Type"] == "application/json"
        )
        response_data = response.json()
        if result_exists:
            assert response_data['result'] is not None
        else:
            assert response_data['result'] is None
        if contained_in_msg:
            if isinstance(contained_in_msg, str):
                assert contained_in_msg in response_data['message']
            elif isinstance(contained_in_msg, list):
>               assert any(x in response_data['message'] for x in contained_in_msg)
E               AssertionError
```

## Task

Figure out if this is indeed flaky. If yes fix it.

At least add a nicer errormessage to be able to find out what the actual response message from the exchange was there.

--- a/rotkehlchen/tests/utils/api.py
+++ b/rotkehlchen/tests/utils/api.py
@@ -96,7 +96,8 @@ def assert_error_response(
         if isinstance(contained_in_msg, str):
             assert contained_in_msg in response_data['message']
         elif isinstance(contained_in_msg, list):
-            assert any(x in response_data['message'] for x in contained_in_msg)
+            msg = f'Response: {response_data["message"]} does not match what we expected'
+            assert any(x in response_data['message'] for x in contained_in_msg), msg
 
 
 def assert_ok_async_response(response: requests.Response) -> int:

